### ISSUE TITLE
range component not accessible

### ISSUE THREAD

**Issue by brettatoms** [ionic-team/ionic#10256]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[x] **2.x**

**I'm submitting a ...**  (check one with "x")
[ ] bug report
[x] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
The Range component doesn't read out range values correctly using Apple's Voice Over.  It also doesn't indicate whether the slide is vertical or horizontal.  I think this could be fixed with the following:
 - Add `aria-orientation="horizontal"` to `<ion-range>`
 - Set `aria-valuemax` and `aria-valuemin` on `.range-slider` to their corresponding component values.
- Set `aria-valuenow` on `.range-slider` to the current range value.

**Expected behavior:**
After enabling Voice Over on iOS you should be able to tab the Range component and it tells you the slider orientation is horizontal.  After double tapping the slider you should be able to drag your finger and Voice Over should tell you the current range value.

**Steps to reproduce:**
(see "Expected Behavior")

**Other information:**
This link has good examples on ARAI attributes for sliders:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/slider-2.html

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

Your system information:

Cordova CLI: 6.4.0
Ionic Framework Version: 2.0.0
Ionic CLI Version: 2.1.18
Ionic App Lib Version: 2.1.9
Ionic App Scripts Version: 1.0.0
ios-deploy version: 1.9.0
ios-sim version: 5.0.9
OS: OS X El Capitan
Node Version: v6.9.1
Xcode version: Xcode 8.2.1 Build version 8C1002

**Comment by brettatoms** [comment]
Ok, I see that ARIA attributes are added to `.range-knob-handle` but VoiceOver doesn't seem to use them correctly from that element.  E.g., if I set `aria-orientation="horizontal" on `.range-knob-handle` then VoiceOver still says "Slide up or down with 1 finger" and when I do change the value then VoiceOver doesn't tell me the value of `aria-valuenow`.  Also, if I tap on random points on the slider it reads out some other number other than the current range value (I'm not sure where the number is coming from).
